video/out/opengl/egl_helpers.c

```diff
@@ -120,22 +156,43 @@ static bool create_context(struct ra_ctx *ctx, EGLDisplay display,
         name = "GLES 2.x +";
     }
 
+    const char *egl_exts = eglQueryString(display, EGL_EXTENSIONS);
+
     MP_VERBOSE(ctx, "Trying to create %s context.\n", name);
 
     if (!eglBindAPI(api)) {
         MP_VERBOSE(ctx, "Could not bind API!\n");
         return false;
     }
 
+    bool request_float_fmt = opts->output_format >> 24;
```

Comment:
I think you forgot this one
```suggestion
    bool request_float_fmt = (opts->output_format & FLOAT_FORMAT);
```

Reply:
The effect is the same.

There are `>> 18`, `>> 12`, and `>> 6` in the following lines. I think the current one looks more consistent in the term of code style.

Reply:
I know.
In my thoughts the point of using a define for the float indicator was to reuse it here, instead of scattering magic numbers over the source file.